nixos/modules/services/web-apps/shiori.nix

```diff
@@ -8,7 +8,12 @@ in {
     services.shiori = {
       enable = mkEnableOption "Shiori simple bookmarks manager";
 
-      package = mkPackageOption pkgs "shiori" { };
+      package = mkOption {
+        type = types.package;
+        default = pkgs.shiori;
+        defaultText = literalExpression "pkgs.shiori";
+        description = "The Shiori package to use.";
```

Comment:
Is this not the same as mkPackageOption?

Reply:
Oh, I based this chance on the most resent commit on unstable, I figured there was a reason for it, but I read the docu and indeed that's the better option!